### ISSUE TITLE
add documentation how to run custom scripts for BIDS conversion

### ISSUE THREAD

**Issue by pvavra** [psychoinformatics-de/datalad-hirni#134]
Many behavioral tasks create log files which are not BIDS-conforming. 

The documentation mentions that custom procedures can be written and referenced in the `studyspec.json` file and automatically executed by the `spec2bids` conversion step. 

An actual step-by-step example would be helpful ([as mentioned in the docs](http://docs.datalad.org/projects/hirni/en/latest/customization.html)). 

My suggestion would be to use a simple `.csv` to `.tsv` file-format conversion script as an example (the `events.tsv` file might be used for that).

By including this, it might be more clear how to include such custom procedures (and where to put them). 

Might be related to previous issue #49, but I guess the overall architecture of hirni has changed since that got closed..

**Comment by bpoldrack** [comment]
True. One thing to point out is, that you can have a closer look into the toolbox. If you ran `datalad run-procedure --discover` on it, you'll find - say the `copy-converter` - and look at its code and have  a look at the toolbox' `.datalad/config` to see how it's referred to in there as well as the actual `studyspec.json` that is used in the respective demo. However, that's just if you want to do some exploration on your own. I agree - this clearly needs documentation.

**Comment by pvavra** [comment]
For reference: 

the datalad-handbook has a [section](http://handbook.datalad.org/en/latest/basics/101-124-procedures.html) on what is required to create a custom procedure and make datalad aware of it (the relevant part is in the box "Find out more: Write your own procedures").

the config referencing the hirni procedures mentioned above is located in `code/hirni-toolbox/.datalad/config` - relative to the dataset where `datalad run-procedure cfg_hirni` was run.

**Comment by pvavra** [comment]
In addressing the issue #136 using small scripts, I also wrote little wrappers that hook up as a `datalad run-procedures`. c.f. [here](https://github.com/pvavra/hirni_addons)

**Comment by bpoldrack** [comment]
@pvavra : Nice! Left a comment there: https://github.com/pvavra/hirni_addons/commit/9415a89b66082bac88c44b62d9732bbf03739639#r36332030
